Commands/Public/room.js

```diff
@@ -0,0 +1,195 @@
+const ArgParser = require("../../Modules/MessageUtils/Parser");
+const moment = require("moment");
+
+module.exports = async ({ Constants: { Colors, Text }, client }, { serverDocument, serverQueryDocument }, msg, commandData) => {
+	const roomDocument = serverDocument.config.room_data.id(msg.channel.id);
+	if (roomDocument) {
+		const question = await msg.send({
+			embed: {
+				color: Colors.PROMPT,
+				description: `This room was created ${moment(roomDocument.created_timestamp).fromNow()}. Would you like to delete it?`,
+			},
+			footer: {
+				text: "You have 1 minute to respond.",
+			},
+		});
+		let response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+		if (response) {
+			try {
+				await response.delete();
+			} catch (err) {
+				// No-op
+			}
+		}
+		if (response && configJS.yesStrings.includes(response.content.toLowerCase().trim())) {
+			let success = true;
+			await msg.channel.delete(`Room Management | Command issued by ${msg.author.tag}`).catch(err => {
+				success = false;
+				winston.debug(`Failed to delete room '${msg.channel.name}' on server '${msg.guild.name}'`, { svrid: msg.guild.id, chid: msg.channel.id, err });
+				msg.send({
+					embed: {
+						color: Colors.LIGHT_ERR,
+						title: "Failed to delete room!",
+						description: "I might be missing sufficient permissions!",
+					},
+				});
+			});
+			if (success) serverDocument.query.id("config.room_data", msg.channel.id).remove();
+		} else {
+			await question.edit({
+				embed: {
+					color: Colors.PROMPT,
+					title: "I'll keep the room intact 😅",
+					description: "Do you want to add any members to this channel?",
```

Comment:
This should not be here at all. If I'm asked to delete the room, I don't expect another `but do you want to xxx`, but rather a simple yes-no

Reply:
At least propose how to handle this differently, where would users otherwise manage members of rooms? I can’t do anything with this as is

Reply:
add new type, `add` or `remove` to add or remove members

Reply:
That'd require quite the refactor. Don't really have time for that currently. It's acceptable as-is. Can be refactored in a later update